### DISCUSSION TITLE
Linter disabled

### DISCUSSION THREAD

**Discussion by lepresk** [nylo-core/nylo#115] [General]
Hello,

I've just noticed that the Flutter linter is completely disabled in this base template. Is there a specific reason for this? If not, I'd be happy to submit a pull request to enable it.

Looking forward to your insights.

**Comment by agordn52** [comment]
Thanks for the [PR](https://github.com/nylo-core/nylo/pull/116) on this 👍

**Reply by lepresk** [reply]
Great, you'are most welcome
